docs/050-breaking-changes.rst

```diff
@@ -248,6 +248,8 @@ Variables
 
 * Fixed-size arrays with a length of zero are now disallowed.
 
+* The ``enum`` type is now allowed in interfaces.
```

Comment:
Please keep this (the next commit removes it), but perhaps rephrase it (enum types in interfaces were always supported) to something like: Enums can now be defined in interfaces.

Reply:
Ah but actually this is not really the right place ("derecated elements")...

Reply:
ok, I ended up reverting it because it wasnt actually a breaking change, just a new feature. No ones old contract were gonna break because of the change.

Reply:
You are right, that actually is true, we do not list new features here.